src/Components/Commodity/CommodityCard.test.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import CommodityCard from './CommodityCard';
+
+describe('CommodityCard', () => {
+  const defaultProps = {
+    symbol: 'ABC',
+    name: 'Test Commodity',
+    currency: 'USD',
+    stockExchange: 'NYSE',
+  };
+
+  test('renders the card with correct props', () => {
+    render(
+      <MemoryRouter>
+        <CommodityCard {...defaultProps} />
+      </MemoryRouter>,
+    );
+
+    const productElement = screen.getByText(/Product: Test Commodity/i);
+    const currencyElement = screen.getByText(/Trading Currency: USD/i);
+    const stockExchangeElement = screen.getByText(/Stock Exchange: NYSE/i);
```

Comment:
Good job working on the project requirements but i noticed you disabled eslint in this file it would be nice if you fixed this by enabling eslint because it is against microverse policy to disable linters because linters help us write clean code.

Reply:
@KayLemba 
Tank You for the review,
Issue Resolved